gcc.yaml

```diff
@@ -113,9 +127,10 @@ subpackages:
   - name: "libquadmath"
     description: "128-bit math library provided by GCC"
     pipeline:
-      - runs: |
+      - if: ${{build.arch}} == 'x86_64'
```

Comment:
Could the entire subpackage be arch conditional rather than an empty package on aarch64? It doesn't look like melange subpackage supports target-architecture perhaps it should?

https://github.com/chainguard-dev/melange/blob/e855ba4a69a3256aa04754dd96906f222d88cbbd/pkg/config/config.go#L358

Reply:
```
2024-07-15T12:46:39.8424616Z 2024/07/15 12:46:39 INFO skipping subpackage libquadmath because ${{build.arch}} == 'x86_64' == false
```

Looks like substitutions are not done correctly.

Reply:
https://github.com/chainguard-dev/melange/issues/1377

Reply:
:cry:  You want to just leave it as it was and put in a link?